### PR TITLE
Added freesolv builder.

### PR DESCRIPTION
**Pull request by kyleabeauchamp**

### CODE REVIEW

File: code/rebuild_freesolv.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Use the gaff2xml wrappers of OpenEye and Antechamber to rebuild input files
+for the FreeSolv database.
+"""
+
+import pickle
+import os
+import glob
+import gaff2xml
+
+FREESOLV_PATH = "/home/kyleb/src/choderalab/FreeSolv/"
```
Comment (pgrinaway) [inline code/rebuild_freesolv.py]: Maybe get rid of a hard-coded FreeSolv path?

**Comment by kyleabeauchamp** [comment]
done

**Comment by pgrinaway** [comment]
+1

**Comment by pgrinaway** [comment]
I think this is good to merge

**Pull request merged**